README.md

```diff
@@ -226,7 +226,11 @@ And you will now get the desired behavior.
 
 This SDK allows you to receive interaction events, for your analytics via the `onEvent` method. At the moment the SDK supports:
 
- - `open` event, which fires when hCaptcha is opened and a challenge is visible to an app user
+ - `open`  fires when hCaptcha is opened and a challenge is visible to an app user
+ - `expired` fires when the passcode response expires and the user must re-verify
+ - `challengeExpired` fires when the user display of a challenge times out with no answer
+ - `close` fires when the user dismisses a challenge.
+ - `error` fires when any error happens during a challenge verification. Note: This event is not for error handling but only for analytics purposes. For error handling please check the `validate` API call
```

Comment:
```suggestion
 - `error` fires when any error happens during a challenge verification, for example a transient network error. Note: This event is not for error handling but only for analytics purposes. For error handling please check the `validate` API call
```

.. don't we need to document the error message passed up?

Reply:
Good point, added information about how actual error can be obtained in `onEvent`